aequilibrae/project/network/node.py

```diff
@@ -0,0 +1,121 @@
+from .safe_class import SafeClass
+from aequilibrae.project.database_connection import database_connection
+from aequilibrae import logger
+
+
+class Node(SafeClass):
+    """A Node object represents a single record in the *nodes* table
+
+    ::
+
+        from aequilibrae import Project
+        from shapely.geometry import Point
+
+        proj = Project()
+        proj.open('path/to/project/folder')
+
+        all_nodes = proj.network.nodes
+
+        # We can just get one link in specific
+        node1 = all_nodes.get(7890)
+
+        # We can find out which fields exist for the links
+        which_fields_do_we_have = node1.data_fields()
+
+        # And edit each one like this
+        node1.comment = 'This node is important'
+
+        # It success if the node_id already does not exist
+        node1.renumber(998877)
+
+        node.geometry = Point(1,2)
+
+        # We can just save the node
+        node1.save()
+        """
+
+    def __init__(self, dataset):
+        super().__init__(dataset)
+        self.__fields = list(dataset.keys())
+
+    def save(self):
+        """Saves node to database"""
+        conn = database_connection()
+        curr = conn.cursor()
+
+        data = []
+
+        if self.node_id != self.__original__['node_id']:
+            raise ValueError('One cannot change the node_id')
+
+        txts = []
+        for key, val in self.__dict__.items():
+            if key not in self.__original__:
+                continue
+            if val != self.__original__[key]:
+                if key == 'geometry' and val is not None:
+                    data.append(val.wkb)
+                    txts.append('geometry=GeomFromWKB(?, 4326)')
```

Comment:
hard coded SRID?

Reply:
Yes. For now.  There are some triggers that would need to be changed if one wanted to change either the projection or the distance unit (which is meters now).  There will be a change sometime in the future where the API will allow the user to change the unit and to create a project in the desired SRID, but not now...

Reply:
I changed it a little bit to make it easier to refactor it when it is time to make it projection agnostic